examples/effects/testNetwork00.wyv

```diff
@@ -0,0 +1,10 @@
+require stdout
```

Comment:
Since these are really for testing purposes, they should not go in the wyvern examples directory.  We do run some tests on examples, but the purpose of programs in the examples directory is to illustrate some feature of Wyvern or show how to do something.

instead, put test cases in a subdirectory of src/wyvern/tools/tests